book/website/reproducible-research/open/open-hardware.md

```diff
@@ -1,121 +1,389 @@
+
+
+<!--  
+## ⚠️ (Sacha, Moritz, Nico M.) TODO next week (Sacha)
+
+:::info
+/// Nico: I've redacted the contents a bit. The table is hard to edit in this format. Let me paste the link again: https://docs.google.com/spreadsheets/d/1JYgMqKapGX7TDFkFo0ZeGXGMx-m2-svK-o4p6NKt1Ns/edit#gid=564457047
+:::
+
+| Documentation item | Audience | Content | Location |
+| -------- | -------- | -------- | -------- |
+|Project overview| General community| Project definition, scope, audience, licensing, and directions to more specialized documentation. Expectation management: state what information is missing and main caveats. | Front page, README.md| 
+|Communication| General community| Links to communication channels: support platforms ( forum, issues at an online repository), online community platforms (for instance forums, chats, etc.), direct contact information, dates for joining in-person or online events. | Front page, README.md| 
+|Contributions| General community| Description of the process of getting involved. List of possible contributions: donations, investments, beta testing, feature development, collaborations, reporting issues. | Contributions page, CONTRIBUTING.md| 
+|Usage| Users| Intended usage instructions, limitations, other use cases. Troubleshooting. Links to learning resources, maintenance and assembly guides. | Usage guide| 
+|Assembly| Manufacturers, Users | Complete description of: the assembly process, used parts, required knowledge, training, number of people, time, and tools. Links to learning resources. | Assembly guide| 
+|Maintenance| Technical support, Users| Instructions for manintenance operations, stating frequency, tested replacement parts, alternatives, and the required skills. List of maintenance services and learning resources. | Maintenance guide| 
+|Design | Developers, Contributors, Educators| Description of the overall design rationale. List of the main design choices and implications (functional compromises or other decisions). Minimal explanation of the working principles, and links to external learning resources. | Development page, DEVELOPMENT.md| 
+|Technical contributions| Manufacturers, Developers, Technical support| Development roadmap, guidelines, core team and contributors, communication channels, version control system, related works. Description of the typical contribution process. Examples of (un)successful contribution processes. | Contributions page, CONTRIBUTING.md| 
+|Development| Developers| Development tools (IDEs, EDA, or CAD software) and style guidelines. Setup for development and testing. Detailed overview of the hardware modules and software. Specifications of the internal and external hardware interfaces. Tutorials and examples of how to get started. | Development page, DEVELOPMENT.md| 
+
+-->
```

Comment:
```suggestion
<!--  

-->
```

Reply:
We don't need to keep the empty comment, imho. Otherwise lgtm.